examples/covasim_/vaccinating_elderly/README.md

```diff
@@ -13,6 +13,9 @@ four test cases: one focusing on each of the four previously mentioned outputs.
 
 Further details are provided in Section 5.3 (Prioritising the elderly for vaccination) of the paper.
 
+Note: this version of the example utilises the observational data collector, which was previously simulated using the 
+experimental data collector, in order to save computation time.
```

Comment:
The experimental data collector is being removed from the framework as it is more logical to separate the software execution and testing. It may be worth rephrasing this to avoid future confusion